deploy/cr/auth/cr_minimal_with_auth.yaml

```diff
@@ -4,5 +4,7 @@ metadata:
   name: example-infinispan
 spec:
   replicas: 2
+  service:
+    type: DataGrid
```

Comment:
better to create a new file under `deploy/cr/datagrid` instead of changing this

Reply:
Yes, sure, will create a new one with DataGrid type